cpu/stm32/Makefile.dep

```diff
@@ -4,7 +4,7 @@
 USEMODULE += periph stm32_clk stm32_vectors
 
 ifneq (,$(filter periph_usbdev,$(FEATURES_USED)))
-  ifeq (,$(filter f3 wb,$(CPU_FAM)))
+  ifeq (,$(filter f1 f3 wb,$(CPU_FAM)))
```

Comment:
I think we have to split it for F1 family, STM32F105xx and STM32F107xx use USB OTG IP core. We could merge it for now as is.